data/migrations/20220222110357_client_schema.js

```diff
@@ -0,0 +1,171 @@
+// I didn't make primary key and foregin key, becasue once we made these key we can't change the schema easily (if you want to know more about foregin key => https://github.com/github/gh-ost/issues/331#issuecomment-266027731)
+
+exports.up = function (knex) {
+  return knex.schema
+    .createTable('locations', (tbl) => {
+      tbl.increments().notNullable().unique();
+      tbl.integer('household_id').notNullable();
+      tbl.string('type');
+      tbl.string('name');
+      tbl.string('latlong');
+      tbl.string('address1');
+      tbl.string('address2');
+      tbl.string('city');
+      tbl.string('state');
+      tbl.string('zip');
+      tbl.datetime('deleted_at');
+      tbl.datetime('created_at').defaultTo(knex.fn.now());
+    })
+    .createTable('households', (tbl) => {
+      //We will verify to stakeholder next week!
+      tbl.increments().notNullable().unique();
+      tbl.string('name').notNullable();
+      tbl.datetime('created_at').defaultTo(knex.fn.now());
+    })
+
+    .createTable('clients', (tbl) => {
+      tbl.increments().notNullable().unique();
+      tbl.integer('household_id').notNullable();
+      tbl.string('first_name').notNullable();
+      tbl.string('last_name').notNullable();
+      tbl.string('ssn', 9).notNullable();
+      tbl.boolean('is_hoh');
+      tbl.string('relation');
+      // tbl.string('education_level');
+      tbl.integer('gender_id');
+      tbl.integer('race_id');
+      tbl.integer('ethnicity_id');
+      tbl.integer('phone_numbers_id');
```

Comment:
You won't need these 3 ids in lines 38, 39, 40 if you put a client_id in the phone_numbers, email_addresses, and education_histories tables (see your schema design). By putting client_id in those 3 tables, each client will be able to have multiple phone numbers, email addresses, and education_histories associated with their case file

Reply:
Ahh, that's right! however, Jake told us that we should not be using PKs/FKs in our schema, and we should reference other tables with their id from the clients table. However, I am also confused about this, because other tables don't work like this (like your notes table) and I think we should talk to Jake/Derek to confirm!

Thank you so much Shane!